### ISSUE TITLE
Different folder for coveraged test and test script will not produce coverage result

### ISSUE THREAD

**Issue by fakhrulhilal** [rfennell/vNextBuild#330]
### VSTS Extension you are using

Pester Test Runner from Black Marble

### Where are you running it?

VSTS

### Version of Extension/Task

7.1.6

#### Actual

The extension should produce coverage report
```
Code coverage report:
Covered 88,24% of 136 analyzed Commands in 2 Files.
Missed commands:

File                   Function                  Line Command
----                   --------                  ---- -------
ScriptUnderTest.ps1    Invoke-VerboseCommand       33 $Variable
```

#### Actual

`[warning]No PowerShell files found under [D:\Temp\vsts\4\s\tests] to analyse for code coverage.`

### Steps to reproduce the problem

It's common to separate the source and test folder. I manage this plugin as follows:
![pester vsts plugin](https://user-images.githubusercontent.com/1755735/40582226-4fcd9d9c-6199-11e8-8ed3-f5d79f0cd681.jpg)
Looking at the source, I guess it's just caused by typo for getting the parameter value
```powershell
if ($CodeCoverageOutputFile -and (Get-Module Pester).Version -ge '4.0.4') {
    if (-not $PSBoundParameters.ContainsKey('CodeCoverageFiles')) {
        $CodeCoverageFolder = $scriptFolder
    }
}
```
It's supposed to be `CodeCoverageFolder`.

### Workaround

Setting the **Scripts Folder or Script** to the parent folder containing both **src** and **tests** folder will solve the problem. But it might test all powershell scripts from unexpected folder, f.e. node_modules, packages, etc. When creating VSTS plugin, we will use [`tfx-cli`](https://docs.microsoft.com/en-us/vsts/extend/publish/command-line?view=vsts) to build the extension. Unfortunately, it bring sample powershell script that will be included in code coverage. This is just one example of the cost.

**Comment by ChrisLGardner** [comment]
I'm working on a few other fixes to this extension so I'll add this in as well. I'm hoping to get it released today or tomorrow.
________________________________

**Comment by ChrisLGardner** [comment]
Released as part of 7.2.1
